CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController.swift

```diff
@@ -109,7 +109,6 @@ public final class Camera2ViewController: UIViewController, CameraScreen {
 
     private func setupView() {
         edgesForExtendedLayout = []
-        view.backgroundColor = giniConfiguration.cameraContainerViewBackgroundColor.uiColor()
```

Comment:
[Clarification] If we remove this Configuration we need to use a color from `GiniColors` palette instead.

Reply:
This backgroundColor is not visible anywhere because of the CameraPreviewVC is above this.